docs/en/observability/apm-alerts.asciidoc

```diff
@@ -1,9 +1,11 @@
-[[apm-alerts]]
-=== Alerts and rules
+////
+TO DO:
+Make sure this aligns with the structure of other
+rule docs in this section.
+////
```

Comment:
I'll follow up on this in https://github.com/elastic/observability-docs/issues/4040.